FreshRSS/FreshRSS
# Improved: Extension manager: new style

Before:
![grafik](https://user-images.githubusercontent.com/1645099/151701572-9e0df7cf-6534-4f5f-a2a1-9cec6d03a684.png)

It has some issues:

1. on wide screens the text is far away from the buttons (see screenshot on a not very wide screen)
2. on mobile screen the text overlaps the buttons (see next screenshot)
3. the middle button is a status button. The status is less clear (the text disable/enable is not the status. It is the action when the user clicks on it).
4. the config button (cog icon) is in most screens in FreshRSS very clear. It does not need the text "manage"
5. the remove button is not very often used but very prominent placed, but not in the extension manage slider

![grafik](https://user-images.githubusercontent.com/1645099/151701631-53f8436b-c1eb-48cf-8861-f5f29e6bccc8.png)

After:
![grafik](https://user-images.githubusercontent.com/1645099/151702535-a44e3dda-c58d-4968-a507-a05119508304.png)

1. fixed. Text is always close to the buttons
2. fixed. No text overlap anymore
3. prominent status (I call it "switch"). Status is very clear.
4. simple manage/config button
5. remove button is removed/moved to the config slider (see screenshot below)

![grafik](https://user-images.githubusercontent.com/1645099/151702553-99cecd57-a113-4880-9468-0ef762ee1532.png)

Other themes:
Alternative dark theme
![grafik](https://user-images.githubusercontent.com/1645099/151702609-1402828c-f043-4f54-8d9c-a08d6dbad08a.png)

Dark theme
![grafik](https://user-images.githubusercontent.com/1645099/151702619-f90fbb1b-dbfa-4c03-84af-dc798ff9f645.png)



Changes proposed in this pull request:

- better HTML
- a lot of CSS

How to test the feature manually:

1. go to the extension configuration
2. see also the nice mouse hover on the switches


Pull request checklist:

- [x] CSS: RTL
- [x] CSS: styling the other themes
- [x] solved the question: Show system extensions to user?
